src/app/storage/duck/operations.ts

```diff
@@ -204,15 +214,10 @@ function* fetchStorageGenerator() {
   const state = yield select();
   const client: IClusterClient = ClientFactory.hostCluster(state);
   const resource = new MigResource(MigResourceKind.MigStorage, state.migMeta.namespace);
-  try {
-    let storageList = yield client.list(resource);
-    storageList = yield storageList.data.items;
-    const refs = yield Promise.all(fetchMigStorageRefs(client, state.migMeta, storageList));
-    const groupedStorages = groupStorages(storageList, refs);
-    return { updatedStorages: groupedStorages, isSuccessful: true };
-  } catch (e) {
```

Comment:
Within generator functions we need to handle the error case by using throw err in a catch block to stop the generator.